pathml/preprocessing/transforms.py

```diff
@@ -1441,12 +1461,13 @@ def F(self, tile):
         counts = anndata.AnnData(
             X=X,
             obs=[
-                tuple([i + tile.coords[0], j + tile.coords[1]])
+                tuple([i + coords_offset[0], j + coords_offset[1]])
```

Comment:
It would be great if you could make a comment specifying whats happening with 
`i + coords_offset[0]` and `j + coords_offset[1]`. Again not a necessary thing but for readability in the future it might help a bit

Reply:
Added, thanks!